src/service/etherscan.ts

```diff
@@ -0,0 +1,15 @@
+import { externalRequest } from "./request";
+
+const ETHERSCAN_API_KEY = "Y2UTKP86K5EYBGZ64YIH6N3IBGNQAJ8M21";
```

Comment:
It would be good to extract the api key in `vue.config`

Reply:
Done as well!